gulpfile.js

```diff
@@ -1,23 +1,120 @@
 var gulp = require('gulp');
+var fs = require('fs');
 var plugins = require('gulp-load-plugins')();
 var del = require('del');
 var glob = require('glob');
 var path = require('path');
+var marked = require('marked');
+var swig = require('swig');
+var through2 = require('through2');
+var promisify = require('es6-promisify');
+var assert = require('assert');
+var mergeStream = require('merge-stream');
+var renderFile = promisify(swig.renderFile);
+var fsWriteFile = promisify(fs.writeFile);
+var fsReadFile = promisify(fs.readFile);
 
-var ignore = ['!./dist', '!./dist/**', '!./node_modules', '!./node_modules/**'];
-var recipes = glob.sync('./!(dist|node_modules)/').map(function toBase(dir) {
+var ignore = ['!./dist', '!./dist', '!./dist/**', '!./node_modules', '!./node_modules/**'];
+var recipeSlugs = glob.sync('./!(dist|node_modules|src)/').map(function toBase(dir) {
   return path.basename(dir);
 });
-var srcRecipes = recipes.map(function makePath(name) {
+var srcRecipes = recipeSlugs.map(function makePath(name) {
   return './' + name + '/**';
 });
-var srcFiles = recipes.map(function makeSourceFile(name) {
-  return './' + name + '/*.js';
-});
+
+var recipes = (function() {
```

Comment:
Yes, it's in my todo above. I started doing it, but it requires a change or two so I was going to wait.